draft-private-access-tokens.md

```diff
@@ -627,52 +727,28 @@ aad = concat(encode(1, keyID),
              encode(2, kemID),
              encode(2, kdfID),
              encode(2, aeadID),
-             encode(len(token_request), token_request))
+             encode(len(blinded_req), blinded_req),
```

Comment:
Should this also pull in the version in the request? (to avoid collisions between future encrypted names (though only really an issue for long lived keys).

Reply:
@chris-wood ?

Reply:
We should probably just bind as much as we can to the ciphertext. I'll update this to include the mapping stuff.